package.json

```diff
@@ -336,6 +341,16 @@
         {
           "command": "azure-api-center.createDeclarativeAgent",
           "when": "(view == apiCenterTreeView && viewItem == azureApiCenterApiVersionDefinitionTreeItem-openapi)"
+        },
+        {
+          "command": "azure-api-center.openInPortal",
+          "when": "view == apiCenterTreeView && viewItem == azureApiCenter",
+          "group": "1@1"
+        },
+        {
+          "command": "azure-api-center.openInPortal",
+          "when": "view == apiCenterTreeView && viewItem == azureApiCenterAzureSubscription",
+          "group": "1@1"
```

Comment:
unnecessary as well

Reply:
since there will be another command that "create api center service" in the next feature, so just set such layout

Reply:
removed